assets/js/registration.js

```diff
@@ -0,0 +1,22 @@
+let timeZone = document.querySelectorAll(".num");
+
+let countDownDate = new Date("July 01, 2021 00:00:00").getTime();
+
+let interval = setInterval(() => {
```

Comment:
`interval` is not good variable name, `intervalId` is better one.

Reply:
fixed